modules/mysql/src/main/java/org/testcontainers/containers/MySQLContainer.java

```diff
@@ -20,6 +20,7 @@
 
     public MySQLContainer() {
         super(IMAGE + ":latest");
+        setCommand("mysqld");
```

Comment:
Don't we also need a similar line in the `public MySQLContainer(String dockerImageName)` constructor?

Reply:
As MySQL container is totally expected to be able to start without specifying command, I have removed manual `setCommand` which duplicates what is already specified in the image.

Reply:
Ha - of course! Makes perfect sense to remove - I wish I knew why it was there in the first place!